guide/creating-your-bot/command-handling.md

```diff
@@ -91,30 +91,12 @@ discord-bot/
 
 Create a new folder named `commands`, which is where you'll store all of your commands.
 
-We'll be using utility methods from the [`@discordjs/builders`](https://github.com/discordjs/discord.js/tree/main/packages/builders) package to build the slash command data, so open your terminal and install it.
+We'll be using utility methods from the package to build the slash command data.
```

Comment:
seems a bit vague as to which package is being talked about perhaps? although this line could also be removed altogether

```suggestion
We'll be using utility methods of the library to build the slash command data.
```